lang/perl5/Portfile

```diff
@@ -95,12 +99,17 @@ foreach {perl5.v perl5.subversion perl5.revision perl5.rmd160 perl5.sha256 perl5
             patchfiles-append \
                             ${perl5.major}/remove-10.3-target-PR126360.patch
         }
-        if {${perl5.major} == 5.26} {
+        if {${perl5.major} >= 5.26} {
             # enable syscall() on Sierra for compatibility with earlier OS versions and perl5.24
             # Apple has deprecated syscall() on Sierra but it is still available
             patchfiles-append \
                             ${perl5.major}/enable-syscall-on-sierra.patch
         }
+        if {${perl5.major} >= 5.28} {
+           patchfiles-append \
+                            ${perl5.major}/0001-perl-127606-use-rpath-to-locate-libperl.dylib-on-OS-.patch
```

Comment:
Was that meant to be 126706?

I would also try if this solves the tests for 5.26.